cypress/integration/checkin/checkin.basic.spec.js

```diff
@@ -0,0 +1,105 @@
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import checkoutActions from '../../support/fragments/checkout/checkout';
+import Users from '../../support/fragments/users/users';
+import InTransitModal from '../../support/fragments/checkin/modals/inTransit';
+
+describe('Check In - Actions ', () => {
+  const userData = {
+    permissions: [permissions.checkinAll.gui,
+      permissions.uiUsersViewLoans.gui,
+      permissions.uiUsersView.gui,
+      permissions.uiInventoryViewInstances.gui,
+      permissions.uiUsersfeefinesCRUD.gui],
+  };
+  const itemData = {
+    barcode: generateItemBarcode(),
+    instanceTitle: `Instance ${getRandomPostfix()}`,
+  };
+
+  before('Create New Item, New User and Check out item', () => {
+    cy.getAdminToken().then(() => {
+      cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => { itemData.instanceTypeId = instanceTypes[0].id; });
+      cy.getHoldingTypes({ limit: 1 }).then((res) => { itemData.holdingTypeId = res[0].id; });
+      cy.getLocations({ limit: 1 }).then((res) => { itemData.locationId = res.id; });
+      cy.getLoanTypes({ limit: 1 }).then((res) => { itemData.loanTypeId = res[0].id; });
+      cy.getMaterialTypes({ limit: 1 }).then((res) => { itemData.materialTypeId = res.id; });
+      ServicePoints.getViaApi({ limit: 1 }).then((servicePoints) => { itemData.servicepointId = servicePoints[0].id; });
+    }).then(() => {
+      InventoryInstances.createFolioInstanceViaApi({ instance: {
+        instanceTypeId: itemData.instanceTypeId,
+        title: itemData.instanceTitle,
+      },
+      holdings: [{
+        holdingsTypeId: itemData.holdingTypeId,
+        permanentLocationId: itemData.locationId,
+      }],
+      items:[{
+        barcode: itemData.barcode,
+        status:  { name: 'Available' },
+        permanentLoanType: { id: itemData.loanTypeId },
+        materialType: { id: itemData.materialTypeId },
+      }] });
+    }).then(specialInstanceIds => {
+      itemData.testInstanceIds = specialInstanceIds;
+    });
+
+    cy.createTempUser(userData.permissions)
+      .then(userProperties => {
+        userData.username = userProperties.username;
+        userData.password = userProperties.password;
+        userData.userId = userProperties.userId;
+        userData.barcode = userProperties.barcode;
+        userData.firstName = userProperties.firstName;
+      })
+      .then(() => {
+        UserEdit.addServicePointViaApi(itemData.servicepointId,
+          userData.userId, itemData.servicepointId);
+
+        const payloadInCheckOutItemRequest = {
+          id: uuid(),
+          itemBarcode: itemData.barcode,
+          loanDate: moment.utc().format(),
+          servicePointId: itemData.servicepointId,
+          userBarcode: userData.barcode,
+        };
+
+        checkoutActions.createItemCheckoutApi(payloadInCheckOutItemRequest);
```

Comment:
use one style. I suggest to start to write fragments from capital letter

Reply:
Done.